dev/DynamicDependency/API/wil_msixdynamicdependency.h

```diff
@@ -10,7 +10,7 @@ typedef unique_any<DLL_DIRECTORY_COOKIE, decltype(&::RemoveDllDirectory), ::Remo
 
 #if defined(MSIXDYNAMICDEPENDENCY_H) && !defined(__WIL_MSIXDYNAMICDEPENDENCY_H)
 #define __WIL_MSIXDYNAMICDEPENDENCY_H
-typedef unique_any<MDD_PACKAGEDEPENDENCY_CONTEXT, decltype(&::MddRemovePackageDependency), ::MddRemovePackageDependency> unique_package_dependency_context;
```

Comment:
Is this new as of 1.0? If so the rename is OK. If not it's a little dangerous since it's a source-breaking change, but I bet nobody uses it directly so the risk is very low.

Reply:
This was new in 1.0

Seemed low enough risk and easily fixed if developers encounter the change and mention in release notes seemed reasonable. Has zero impact on binary or behavior compat.